http/oidc/src/main/java/org/wildfly/security/http/oidc/RequestAuthenticator.java

```diff
@@ -126,4 +206,37 @@ protected void completeAuthentication(OidcRequestAuthenticator oidc) {
         completeOidcAuthentication(principal);
         log.debugv("User ''{0}'' invoking ''{1}'' on client ''{2}''", principal.getName(), facade.getRequest().getURI(), deployment.getResourceName());
     }
+
+    protected void completeAuthentication(BearerTokenRequestAuthenticator bearer) {
+        RefreshableOidcSecurityContext session = new RefreshableOidcSecurityContext(deployment, null, bearer.getTokenString(), bearer.getToken(), null, null, null);
+        final OidcPrincipal<RefreshableOidcSecurityContext> principal = new OidcPrincipal<>(bearer.getToken().getPrincipalName(deployment), session);
+        completeBearerAuthentication(principal);
+        log.debugv("User ''{0}'' invoking ''{1}'' on client ''{2}''", principal.getName(), facade.getRequest().getURI(), deployment.getResourceName());
+    }
+
+    protected boolean isAutodetectedBearerOnly(OidcHttpFacade.Request request) {
```

Comment:
@fjuma Just a minor, the request is unused. The below calls to facade.getRequest() can be substituted for the request passed to the method

Reply:
Good catch! I've updated the method to remove the parameter since it's not needed.